.travis.yml

```diff
@@ -23,6 +23,7 @@ jobs:
 install:
   - sudo apt-get install pandoc # pandoc for jupyter notebooks
   - sudo apt install graphviz # graphviz for class inheritance diagrams in docs
+  - pip install isort # isort for sorting imports
```

Comment:
This dependency should not be necessary. When running `tox -e isort` tox will create a virtual environment and use the dependencies which are listed in `./tox.ini`, which includes isort.

On another note: I think it would be useful to add the isort check to our CI workflow. Right now we are using Github-actions for the linting (Black and pylint), so it would be great if you could add isort to our linting workflow (`.github/workflows/linting.yml`).

Let us know if you have any questions!

Reply:
I set it up, let me know if it's good.

Reply:
The github action looks good to me, thanks!